packages/mobile/src/alert/AlertBanner.test.tsx

```diff
@@ -37,13 +64,13 @@ describe('AlertBanner', () => {
           }}
         />
       )
-      expect(tree).toMatchSnapshot()
+      expect(toJSON()).toMatchSnapshot()
```

Comment:
Why toJSON?

Reply:
I switched to `react-native-testing-library` cause `react-test-renderer` only captured the initial render which was `null`.
`toJSON` is the way to do it with that lib.